extensions/resteasy-classic/rest-client/config/src/main/java/io/quarkus/restclient/config/RestClientConfigRootProvider.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.restclient.config;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class RestClientConfigRootProvider {
```

Comment:
I wouldn't include this, because the config may be injected directly and is just adding an additional layer for little benefit.